.envrc

```diff
@@ -0,0 +1,13 @@
+if ! find_up ./.venv/checkvenv.txt; then
+    python3 -m venv .venv
+    touch ./.venv/checkvenv.txt
+fi
+source ./.venv/bin/activate
+
+if ! find_up ./.venv/spawnpoint.txt; then
+    pip install .
+    pip install '.[devtool]'
+    touch ./.venv/spawnpoint.txt
```

Comment:
I'd call this file in the venv `.direnv` to signify that we touched it vs Python.